Formula/rtptools.rb

```diff
@@ -0,0 +1,31 @@
+class Rtptools < Formula
+  desc "Set of tools for processing RTP data"
+  homepage "https://www.cs.columbia.edu/irt/software/rtptools/"
+  url "https://www.cs.columbia.edu/irt/software/rtptools/download/rtptools-1.21.tar.gz"
+  sha256 "2ddbf6d3a4dbdef26ca85e860f6795264a359d30e24fd63f14a8c4f7ca331830"
+
+  def install
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  test do
+    packet = [
+      0x5a, 0xb1, 0x49, 0x21, 0x00, 0x0d, 0x21, 0xce, 0x7f, 0x00, 0x00, 0x01,
+      0x11, 0xd9, 0x00, 0x00, 0x00, 0x18, 0x00, 0x10, 0x00, 0x00, 0x06, 0x8a,
+      0x80, 0x00, 0xdd, 0x51, 0x32, 0xf1, 0xab, 0xb4, 0xdb, 0x24, 0x9b, 0x07,
+      0x64, 0x4f, 0xda, 0x56
+    ]
+
+    File.open(testpath/"test.rtp", "wb") do |file|
+      file.puts "#!rtpplay1.0 127.0.0.1/55568"
+      file.write packet.pack("c*")
+    end
+
+    output = shell_output("#{bin}/rtpdump -F ascii -f #{testpath}/test.rtp")
+    assert_match(/seq=56657 ts=854698932 ssrc=0xdb249b07/, output)
```

Comment:
Can you explain what this is supposed to be testing? Even if I morph the data in packet to the point that it prints an error, it still passes.

Reply:
An RTP packet consists of a header and payload. If you modify the payload, it isn't going to care. If you modify the header, it is only going to fail if you change the seq, ts or ssrc fields.

The main thing is to test that the rtpdump build was successful, not verify its behaviour.